### PR TITLE
Extract Like component to make it reusable

### LINKED ISSUE #280: [Liking Content] Extract code from <ActionContainer /> to form own <Like /> component
We should create a `<Like />` component that is reusable which contains the code/logic from the `<ActionContainer />` component. 

#### Acceptance Criteria _(Optional)_

* [ ] We should have a reusable `<Like />` component

### PR DESCRIPTION
**Pull request by IsaacHardy**
Fixes #280 

## Creator

- [x] Build relevant tests if any apply
- [x] Ensure there are no new warnings (in app AND in storybook)
- [ ] Upload an animated GIF showcasing the solution (if it applies)
- [x] Set a relevant reviewer
- [x] PR has a relevant title that adheres to our patch-notes/change-log style

## Reviewer

- [ ] Review CI build results
- [ ] Review function and form on iOS
- [ ] Review function and form on Android
- [ ] Review new stories if any apply
- [ ] Review test coverage: Are all new or changed states tested?
- [ ] Review new test logic
- [ ] Review that the PR title adheres to our patch-notes/change-log style

### CODE REVIEW

File: packages/apolloschurchapp/src/ui/LikeButton/index.js [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+              {(createNewInteraction) => (
+                <Like
+                  itemId={itemId}
+                  sessionId={sessionId}
+                  isLiked={isLiked}
+                  operation={isLiked ? 'Unlike' : 'Like'}
+                  toggleLike={async (variables) => {
+                    try {
+                      await createNewInteraction({ variables });
+                      await refetch();
```
Comment (vinnyjth) [inline packages/apolloschurchapp/src/ui/LikeButton/index.js]: Do we still need the refetch? I think we can get rid of this now that we have the `cache.writeQuery` :)

**Comment by vinnyjth** [comment]
Reviewing 👀

File: packages/apolloschurchapp/src/content-single/ActionContainer.js [hunk 2]
```diff
@@ -1,79 +1,22 @@
...
-                )}
-              </Mutation>
-            )}
-          </Query>
-        ) : null
-      }
-    </Query>
+    <LikeButton
+      itemId={itemId}
+      updateLikeEntity={updateLikeEntity}
```
Comment (vinnyjth) [inline packages/apolloschurchapp/src/content-single/ActionContainer.js]: Nit-pick. The Share button can be put anywhere and passed an object to share. The same isn't true with the like button. You need to inject the `updateLikeEntity` function and the `getLikedContentItem` function. What do you think about moving those functions inside the like button component so they don't need to be passed?

File: packages/apolloschurchapp/src/ui/LikeButton/LikeButton.tests.js [hunk 3]
```diff
@@ -0,0 +1,78 @@
+import React from 'react';
```
Comment (vinnyjth) [inline packages/apolloschurchapp/src/ui/LikeButton/LikeButton.tests.js]: TESTS :D 

nice work :)

File: packages/apolloschurchapp/src/ui/LikeButton/index.js [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+          )}
+        </Query>
+      ) : null
+    }
+  </Query>
+);
+
+LikeButton.propTypes = {
+  itemId: PropTypes.string,
+  getLikedContentItem: PropTypes.shape({}),
```
Comment (vinnyjth) [inline packages/apolloschurchapp/src/ui/LikeButton/index.js]: Nit pik - it's annoying to fill these out, but it always helps future readers of this component. We probably won't need them if we move the functions inside this component, which would be nice :D

File: packages/apolloschurchapp/src/ui/LikeButton/index.js (hunk 1 above)
Comment (IsaacHardy) [reply packages/apolloschurchapp/src/ui/LikeButton/index.js]: I swore I removed this... Maybe I'm losing my mind.

**Comment by IsaacHardy** [comment]
@vinnyjth Made updates to comments

File: packages/apolloschurchapp/src/content-single/ActionContainer.js (hunk 2 above)
Comment (vinnyjth) [reply packages/apolloschurchapp/src/content-single/ActionContainer.js]: You're very right, good catch

**Pull request merged**